### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by benmccann**

### CODE REVIEW

**Comment by benmccann** [comment]
Hmm. I can't understand why I'm getting this error. The class looks to be in the jackson jar. Maybe the jar isn't being included for some reason?

        java.lang.ClassNotFoundException: com.fasterxml.jackson.core.JsonFactory

@michel-kraemer do you know what's going on? your expertise here would be most appreciated :-)

File: build.gradle [hunk 1]
```diff
@@ -58,6 +59,10 @@ dependencies {
     integrationTest24 jacksonCore + '2.4.0'
     integrationTest24 jacksonAnnotations + '2.4.0'
     integrationTest24 jacksonDatabind + '2.4.0'
+
+    integrationTest24 jacksonCore + '2.5.0'
```
Comment (michel-kraemer) [inline build.gradle]: Maybe it's because you put `integrationTest24` here instead of `integrationTest25`?

**Comment by michel-kraemer** [comment]
Funny thing. I just looked into my local workspace and noticed that I made the exact same change a couple of weeks ago already (85d5fbcb). I just forgot to push. LOL. I think I can close this one. Sorry for the inconvenience, but thanks for the contribution!

Cheers,
Michel

File: build.gradle (hunk 1 above)
Comment (benmccann) [inline build.gradle]: D'oh! Thanks

**Comment by benmccann** [comment]
Thanks @michel-kraemer !

The README says "Due to some compatibility issues both libraries' major and minor version numbers have to match."  It doesn't look to me like there'd be any issue with using bson4jackson 2.4 with mongojack 2.5. Am I right that it'd be safe or do I need to wait for a new release before using mongojack 2.5?

**Comment by michel-kraemer** [comment]
Yes, it seems 2.4 and 2.5 are really compatible. The note in the README refers to some changes made in 2.2 and 2.3 (AFAIK). You should be good to go.

Cheers,
Michel

**Pull request closed without merging**